lib/assessment.js

```diff
@@ -5,7 +5,15 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  var minimum = arguments[0];
+  for (var i = 0; i < arguments.length; i++) {
+    if (arguments[1] < minimum) {
```

Comment:
This tests whether the 1th index of `arguments` is less than `minimum` each iteration of the loop. This leads to an incorrect result.